Tox Twitter lowercase problem.

**Describe the bug**
Tox Twitter lowercase problem. 

* https://github.com/WDscholia/scholia/actions/runs/2967930634
```
pytest --doctest-modules scholia/query.py 
============================================= test session starts ==============================================
platform linux -- Python 3.8.10, pytest-6.2.1, py-1.10.0, pluggy-0.13.1
rootdir: /home/faan/projects/scholia
collected 33 items                                                                                             

scholia/query.py ............................F....                                                       [100%]

=================================================== FAILURES ===================================================
____________________________________ [doctest] scholia.query.twitter_to_qs _____________________________________
1295         Twitter account identifier
1296 
1297     Returns
1298     -------
1299     qs : list of str
1300         List of strings with Wikidata IDs.
1301 
1302     Examples
1303     --------
1304     >>> twitter_to_qs('utafrith') == ['Q8219']
Expected:
    True
Got:
    False
```

--- a/scholia/query.py
+++ b/scholia/query.py
@@ -1307,7 +1307,7 @@ def twitter_to_qs(twitter):
     """
     # This query matches exact and lowercased version of username
     query = """SELECT DISTINCT ?item
-               WHERE {{ VALUES ?username {{ "{twitter}" "{lower}" }} 
+               WHERE {{ VALUES ?username {{ "{twitter}" "{lower}" }}
                         ?item wdt:P2002 ?username }}""".format(
         twitter=escape_string(twitter), lower=escape_string(twitter).lower()
     )

--- a/scholia/query.py
+++ b/scholia/query.py
@@ -1306,8 +1306,8 @@ def twitter_to_qs(twitter):
 
     """
     # This query matches exact and lowercased version of username
-    query = """select ?item
-               where {{ VALUES ?username {{ "{twitter}" "{lower}" }} 
+    query = """SELECT DISTINCT ?item
+               WHERE {{ VALUES ?username {{ "{twitter}" "{lower}" }} 
                         ?item wdt:P2002 ?username }}""".format(
         twitter=escape_string(twitter), lower=escape_string(twitter).lower()
     )

